store/postgres/migrations/000016_update_replay_table_add_config_column.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE replay DROP IF EXISTS config;
```

Comment:
Why is replay table being modified in this PR?

Reply:
it should not. this is a new migration file that is being added in other PR (not merged yet), and we are adding 000017 in this PR. this does not belong to this PR

Reply:
This is still not removed?